### ISSUE TITLE
Plug/unplug external monitor black desktop

### ISSUE THREAD

**Issue by gobonja** [wsdfhjxc/kwin-scripts#4]
If script is enabled - plug/unplug external monitor causes black desktop on every virtual screen except one. 
Right click doesn't work on desktops with black background.

If script is disabled before plug/unplug and enabled after everything works fine.

`kquitapp5 plasmashell && kstart5 plasmashell` fixes issue until next plug/unplug.

Here are screenshots:
![1](https://user-images.githubusercontent.com/26535797/106203309-784ca680-61bb-11eb-89ca-cf899039e5ef.png)

![2](https://user-images.githubusercontent.com/26535797/106203313-797dd380-61bb-11eb-9a17-a42a2e8e202e.png)

![3](https://user-images.githubusercontent.com/26535797/106203314-7a166a00-61bb-11eb-9248-7f134b3ed06e.png)

Best regards

**Comment by wsdfhjxc** [comment]
I don't have a second monitor available at the moment, but I'll see what I can do. Thanks for reporting, and sorry for trouble.

**Comment by jdillon** [comment]
I found that on reboot the same behavior with all but one virtual desktop is black.  I have to disable the script before every reboot and then enable once the system has rebooted.
I also see the problem with a display coming/going (when the displays input is changed) though additionally I found that the top-panel also seems to go missing until the system is restarted (with the disable/enable script trick).
